src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocNodeImpl.java

```diff
@@ -72,6 +72,20 @@
      */
     private DetailNode parent;
 
+    /**
+     * Reflects if any of the successors of this {@code DetailNode} has property
+     * {@link #unclosedTag} set true.
+     */
+    private Boolean hasUnclosedTags;
+
+    /**
+     * Reflects whether the {@code DetailNode} represents a tag which is not closed.
+     * For example consider input: {@code <p> This is a para}, in which {@code <p>} tag is unclosed.
+     * @see <a href=
+     *     "https://www.w3.org/TR/html51/syntax.html#optional-start-and-end-tags">Optional tags</a>
+     */
+    private Boolean unclosedTag;
```

Comment:
ok, after searching a bit on the web - looks like it's a widely used practice, let's leave `unclosedTag`